### ISSUE TITLE
Option to leave out plural "s" from table name/alias

### ISSUE THREAD

**Issue by sveniu** [restsql/restsql#48]
There's currently a hardcoded "s" appended to table names and aliases, so that something like this:
```
<metadata>
  <database default="mydb" />
  <table name="foo" role="Parent" /> <!-- or with alias="..." -->
</metadata>
```
... ends up as
```
{ "foos": [ ... ] }
```

This is well and good in most cases, but sometimes it ends up looking very odd. Having an option to not append the "s" would be great.

**Comment by bennettellis** [comment]
:+1: for this feature addition

**Comment by restsql** [comment]
Fixed with release 0.8.11
